contrib/terraform/openstack/inventory.tpl

```diff
@@ -0,0 +1,28 @@
+${all_servers}
+
+[etcd]
+${etcd_name}
+
+[kube-master]
+${masters_name}
+
+[kube-node]
+${nodes_name}
+
+
+[k8s-cluster:children]
+kube-master
+kube-node
+
+[k8s-cluster:vars]
+ansible_become=True
+ansible_become_method=sudo
+ansible_user=centos
+
+[etcd:vars]
+ansible_become=True
+ansible_become_method=sudo
+ansible_user=centos
```

Comment:
This may depend on the image ID, which is a variable, so may want to turn this into a variable as well.